src/redis.ts

```diff
@@ -1,13 +1,14 @@
 import { RedisPubSub } from 'graphql-redis-subscriptions';
 import Redis, { RedisOptions } from 'ioredis';
+import { IORedisPool, IORedisPoolOptions } from 'ts-ioredis-pool';
 
 export const redisOptions: RedisOptions = {
   host: process.env.REDIS_HOST,
   port: parseInt(process.env.REDIS_PORT, 10),
   password: process.env.REDIS_PASS,
 };
 
-export const redisClient: Redis.Redis = new Redis(redisOptions);
+// export const redisClient: Redis.Redis = new Redis(redisOptions);
 
 export const redisPubSub = new RedisPubSub({
```

Comment:
Can't we migrate this as well and get rid of `ioredis`?

Reply:
fixed. is it how you wanted it?